ports/raspberrypi/boards/raspberry_pi_pico_w/mpconfigboard.h

```diff
@@ -5,6 +5,7 @@
 #define CIRCUITPY_DIGITALIO_HAVE_INVALID_DRIVE_MODE (1)
 
 #define MICROPY_HW_LED_STATUS   (&pin_CYW0)
+#define MICROPY_PY_COLLECTIONS_DEQUE (1)
```

Comment:
This should probably be promoted up to `mpconfigport.h`. It should be on for all but the smallest builds.

Reply:
Deque is used by wifi, and not all rp2 boards have wifi, so that's why I put it at the board level.

Reply:
See https://github.com/adafruit/circuitpython/pull/6474; we generally want `deque` on; only turning it off on the smallest builds.